torch/nn/modules/instancenorm.py

```diff
@@ -57,7 +57,7 @@ def forward(self, input):
 
 @weak_module
 class InstanceNorm1d(_InstanceNorm):
-    r"""Applies Instance Normalization over a 2D or 3D input (a mini-batch of 1D
+    r"""Applies Instance Normalization over a 3D input (a mini-batch of 1D
```

Comment:
If I try it with a 2D input I get the following error: ValueError: InstanceNorm1d returns 0-filled tensor to 2D tensor.This is because InstanceNorm1d reshapes inputs to(1, N * C, ...) from (N, C,...) and this makesvariances 0.
My Pytorch version is 1.10.1.
Same issue was reported here: https://discuss.pytorch.org/t/instancenorm1d-returns-0-filled-tensor-to-2d-tensor-this-is-because-instancenorm1d-reshapes-inputs-to-1-n-c-from-n-c-and-this-makesvariances-0/131371/2

Reply:
Note that the master branch version supports 2D (unbatched) input and no longer throws the error.

Reply:
Can you please propose a proper change to InstanceNorm1d doc? I don't have that version.